sosw/components/config.py

```diff
@@ -45,6 +45,95 @@
 from sosw.components.dynamo_db import DynamoDbClient
 
 
+class SecretsManager:
+
+    secretsmanager_client = None
+
+
+    def __init__(self, test=False, **kwargs):
+
+        self.test = test
+
+        if not self.test:
+            self.test = True if os.environ.get('STAGE') == 'test' or os.environ.get('autotest') == 'True' else False
+
+
+    def _get_secretsmanager_client(self):
+
+        if self.secretsmanager_client is None:
+            self.secretsmanager_client = boto3.client('secretsmanager')
+
+        return self.secretsmanager_client
+
+
+    def call_boto_secrets_with_pagination(self, f, **kwargs):
+        """
+        Invoke SecretsManager functions with the ability to paginate results.
+
+        :param str f:           SecretsManager function to invoke.
+        :param object kwargs:   Keyword arguments for the function to invoke.
+        :rtype list
+        :return:                List of paginated responses.
```

Comment:
Describe better please. For example say that the response is iterable.
But it's not really always a "List of paginated responses"

Reply:
fixed